### PR TITLE
Made minor changes according to Adi's comments

### PR DESCRIPTION
**Pull request by shenkw1**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -16,6 +16,8 @@
 
 bot = commands.Bot(command_prefix = "-")
 
+ROOT_URL = "https://esports-api.lolesports.com/persisted/gw"
```
Comment (adityaxdiwakar) [inline bot.py:19]: Great job!

File: bot.py [hunk 2]
```diff
@@ -36,8 +38,7 @@ async def ping(ctx):
 # Outputs all the supported leagues, organized by region in embed menu
 @bot.command(help='Returns supported leagues')
 async def list(ctx):
-    response = requests.get("https://esports-api.lolesports.com/persisted/gw/getLeagues?hl=en-US", headers=headers)
-    print(response.status_code)
+    response = requests.get(ROOT_URL + "getLeagues?hl=en-US", headers=headers)
```
Comment (adityaxdiwakar) [inline bot.py:41]: Perfect.

File: bot.py [hunk 3]
```diff
@@ -64,21 +65,11 @@ async def list(ctx):
...
-""" # Get schedule command
-# Outputs the weekly schedule for the specified league
-# arg = league abbreviation (LCS, LEC, etc)
-@bot.command(help = 'Returns schedule of specified league')
-async def sched(ctx, league):
-    if isinstance(error, commands.MissingRequiredArgument):
-        await ctx.channel.send("Please specify which league you would like to see the schedule of (``-list`` for a list of leagues)")
-    else:
-        await ctx.channel.send("Placeholder, will be pulling from lolesports API") """
-
```
Comment (adityaxdiwakar) [inline bot.py:83]: Yea, this was another minor thing. Typically, you should save additional features for later until you've merged a big feature into the master branch. If you want to work on features concurrently, you can use multiple branches.

**Review by adityaxdiwakar — APPROVED** [review]
As always, amazing job 💓

File: bot.py (hunk 3 above)
Comment (shenkw1) [reply bot.py:83]: ye I accidentally started to work on the sched command before the list command and I didn't want to delete it so i just commented it out lmao

**Pull request merged**